Public/Src/App/Bxl/Args.cs

```diff
@@ -1129,6 +1129,9 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                         OptionHandlerFactory.CreateBoolOption(
                             "vs",
                             sign => ideConfiguration.IsEnabled = sign),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "vsNew", // temporary undocumented option for enabling new VS solution generation

```

Comment:
>New [](start = 31, length = 3)

Is there some label for the new strategy? There may come a day where this is years old and not justifying of "new" anymore .:smile: If we're unimaginative maybe just v2

Reply:
I hope this is not going to survive for a whole year.  I thought I'd remove this in a month or so